doc/overview.md

```diff
@@ -7,11 +7,11 @@ Here you’ll learn more about deployment options, environments provided, and ho
 ## Deployment options
 
 You can use APIcast hosted or self-managed, in both cases, it needs connection to the rest of the 3scale API management platform:
-- **<a href="https://support.3scale.net/docs/deployment-options/apicast-cloud-gateway">APIcast hosted</a>**: 3scale hosts the gateway in the cloud. The API gateway is already deployed for you and it's limited to 50,000 calls per day.
-- **APIcast self-managed**: You can deploy APIcast wherever you want. To do so, download the json configuration file from API > Integration > Production or fetch it using the Account management API. The self-managed mode is the intended mode of operation for production environments. Here are a few recommended options to deploy your API gateway:
-  - 'Local' deploy: Install dependencies (check out the [Tools and dependencies info](https://github.com/3scale/apicast#tools-and-dependencies) and get the 'v2.0.0-rc1' tagged version of APIcast (or latest release published).
+- **Embedded APIcast**: Two APIcast gateways (staging and production) come by default with the 3scale API Management installation. They come pre-configured and ready to use out-of-the-box.

```

Comment:
```suggestion
- **Embedded APIcast**: A 3scale API Management installation includes two default APIcast gateways, staging and production. These gateways come preconfigured and ready for immediate use.

```

Reply:
Copied from https://access.redhat.com/documentation/en-us/red_hat_3scale_api_management/2.13/html/installing_3scale/installing-apicast#apicast-deployment-options

It should be updated there as well ;)